fe/fe-common/src/main/java/org/apache/doris/common/Config.java

```diff
@@ -1717,7 +1717,7 @@ public class Config extends ConfigBase {
      * If set to TRUE, FE will convert date/datetime to datev2/datetimev2(0) automatically.
      */
     @ConfField(mutable = true, masterOnly = true)
-    public static boolean enable_date_conversion = false;
+    public static boolean enable_date_conversion = true;
```

Comment:
Will the datev2/datetimev2 be enabled by default?

Reply:
no, this is just for test, will remove it soon.